UPGRADING.rst

```diff
@@ -10,6 +10,14 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#3448 Make snapshot location configurable
```

Comment:
2 new lines before a heading.

Reply:
Does the heading have to match the commit title? I don't see anything within the contributing.rst..... 🤔